examples/mobile/HomeApp/js/app.js

```diff
@@ -321,11 +374,13 @@ import Storage from "./clipping-storage.js";
 			app.webClipsList.forEach((webClip) => {
 				if (webClip.manifest && webClip.manifest.cardType === "control") {
 					if (webClip.isSelected) {
-						homeApp.addControlCard({
-							title: webClip.manifest.description,
-							href: "#nowa-karta",
-							icon: "images/Icon.png"}
-						);
+						if (!document.querySelector("[data-title='" + webClip.manifest.description + "']")) {
+							homeApp.addControlCard({
+								title: webClip.manifest.description,
+								href: "#nowa-karta",
```

Comment:
"nowa-karta" can we change this to English?

Reply:
I'll change this.

Reply:
Fixed.